signing/signing.go

```diff
@@ -202,15 +198,76 @@ func postResults(endpoint *url.URL, payload []byte) error {
 	return nil
 }
 
-func extractTlogIndex(bundlePath string) (int64, error) {
-	contents, err := os.ReadFile(bundlePath)
+func extractTlogIndex(bundle *protobundle.Bundle) (int64, error) {
+	// what does multiple tlog entries mean?
```

Comment:
You would have multiple entries if you submitted the record to multiple logs, for instance if `root.SelectServices()` for rekor returned multiple logs from the signing config.